echopype/tests/test_core.py

```diff
@@ -63,3 +66,21 @@ def test_file_extension_validation_should_fail(
         raise ValueError(
             f"\"{ext}\" should have been rejected for sonar model {sonar_model}"
         )
+
+
+def test_init_ep_dir(monkeypatch):
+    temp_zarr_dir = tempfile.TemporaryDirectory()
+    zarr_file_name = Path(temp_zarr_dir.name) / ".echopype"
```

Comment:
Why refer to zarr in these path names? I realize that Zarr writing will be one of the main usages for the user temp directory  you're proposing, but that's not central to this test.